apps/mobile/app/_layout.tsx

```diff
@@ -1,5 +1,6 @@
 import { Stack } from 'expo-router'
 import 'react-native-reanimated'
+
```

Comment:
If your IDE adding those empty lines? I saw something like this in a different file as well.